show-coordinates-in-multiple-formats/src/main/java/com/esri/arcgismaps/sample/showcoordinatesinmultipleformats/screens/MainScreen.kt

```diff
@@ -35,21 +49,60 @@ import com.esri.arcgismaps.sample.showcoordinatesinmultipleformats.components.Ma
 fun MainScreen(sampleName: String) {
     // create a ViewModel to handle MapView interactions
     val mapViewModel: MapViewModel = viewModel()
+    // create a map that has the WGS 84 coordinate system and set this into the map
+    val basemapLayer = ArcGISTiledLayer(LocalContext.current.applicationContext.getString(R.string.basemap_url))
+    val arcGISMap = ArcGISMap(Basemap(basemapLayer))
+    // the collection of graphics overlays used by the MapView
+    val graphicsOverlayCollection = rememberGraphicsOverlayCollection()
+    // graphics overlay for the MapView to draw the graphics
+    val graphicsOverlay = remember { GraphicsOverlay() }
+    // set up a graphic to indicate where the coordinates relate to, with an initial location
+    val initialPoint = Point(0.0, 0.0, SpatialReference.wgs84())
+
+    val coordinateLocation = Graphic(
+        geometry = initialPoint,
+        symbol = SimpleMarkerSymbol(
+            style = SimpleMarkerSymbolStyle.Cross,
+            color = Color.fromRgba(255, 255, 0, 255),
+            size = 20f
+        )
+    )
+    graphicsOverlay.graphics.add(coordinateLocation)
+    // update the coordinate notations using the initial point
+    coordinateLocation.geometry = initialPoint
+    mapViewModel.toCoordinateNotationFromPoint(initialPoint)
 
     Scaffold(
         topBar = { SampleTopAppBar(title = sampleName) },
         content = {
             Column(
-                modifier = Modifier.fillMaxSize().padding(it)
+                modifier = Modifier
+                    .fillMaxSize()
+                    .padding(it)
             ) {
                 // layout to display the coordinate text fields.
                 CoordinatesLayout(mapViewModel = mapViewModel)
-                // composable function that wraps the MapView
-                ComposeMapView(
+                MapView(
                     modifier = Modifier.fillMaxSize(),
-                    mapViewModel = mapViewModel,
-                    onSingleTap = { singleTapConfirmedEvent ->
-                        mapViewModel.onMapTapped(singleTapConfirmedEvent.mapPoint)
+                    arcGISMap = arcGISMap,
+                    graphicsOverlays = graphicsOverlayCollection.apply {
+                        this.add(graphicsOverlay)
+                    },
```

Comment:
I think simple assignment properties here more clear, and we can move the `apply` block away from the Compose call and add it to the line the val is declared. 

```kotlin
    // graphics overlay for the MapView to draw the graphics
    val graphicsOverlay = remember { GraphicsOverlay() }
    // the collection of graphics overlays used by the MapView
    val graphicsOverlayCollection = rememberGraphicsOverlayCollection().apply {
        add(graphicsOverlay)
    }
```

```suggestion
                    graphicsOverlays = graphicsOverlayCollection,
```

Reply:
This fixed the issue with showMessageDialog not showing and app crashing with error (GO object already owned)
I think everytime MapView composable was called, we were adding the same GO object to the GO collection, without clearing it. 
but moving the apply block in declaration fixes this. Thanks!